lib/Data/Aeson/Combinators/Decode.hs

```diff
@@ -71,6 +71,10 @@ module Data.Aeson.Combinators.Decode (
 -- $jsonpath
   , element
   , path
+-- *** Dealing With Failure
+  , jsonMaybe
+  , jsonEither
```

Comment:
These two are cool! Let me just think about a naming - This library encourages qualified imports so I don't think we should do Apps Hungarian prefixing for these two.

Reply:
Thanks! I originally tried to name `jsonMaybe` just `maybe`, but that conflicts with the `maybe` function from prelude and I'm not sure what to do about that.

Reply:
you can do `Import Prelude hiding (maybe)` I believe

Reply:
Maybe I'm not doing it right, but from what I can tell, even though `import Predule hiding (maybe)` does allow me to define a new `maybe` and use it, I still cannot export it. I still get `Ambiguous occurrence 'maybe'`.

Reply:
Never mind, I was doing it wrong. I added a new "import Prelude hiding", not realizing there already was one, so in effect nothing was being hidden. Anyway, commit incoming.